src/endpoint/endpoint.js

```diff
@@ -269,6 +270,8 @@ function create_endpoint_handler(init_request_sdk, virtual_hosts, sts, logger) {
 
     /** @type {EndpointHandler} */
     const endpoint_request_handler = (req, res) => {
+        res.setHeader('Server', `NooBaa/${pkg.version}`);
```

Comment:
but what about the other handlers - sts/iam?
can you add it as function in entpoint_utils and call from all handlers?